docker/startup.sh

```diff
@@ -45,4 +45,23 @@ if [ -n "$R_PACKAGES" ]; then
   R -e "install.packages($R_PACKAGES, repos = 'http://cran.us.r-project.org')"
 fi
 
+# Install custom cacerts
+if [ -e /mnt/mesos/sandbox/cacerts ]; then
+	find /usr/lib/jvm -name cacerts -exec cp /mnt/mesos/sandbox/cacerts '{}' \; 
+fi
+
+# Add custom jars 
+if [ -e /mnt/mesos/sandbox/jars ]; then
```

Comment:
Wouldn't it be better do do something like `if ls /mnt/mesos/sandbox/*.jar; then ...`? That way no extra file `/mnt/mesos/sandbox/jars` is necessary